### ISSUE TITLE
autocompletion is bloken (vscode zls)

### ISSUE THREAD

**Issue by profelis** [ryupold/examples-raylib.zig#10]
after this commit https://github.com/ryupold/examples-raylib.zig/commit/abe5e8f3b2a1e4fc5be0944d4b45b9a86bc75ff3
autocompletion is broken, no more autocompletion after raylib(dot).

zls shows hint that module is unknown
```
const raylib = @import("raylib")
(unknown)
```

before this commit raylib variable was hinted as type
```
const raylib = @import("../../raylib/raylib.zig")
(type)
```
and all functions was autocompleted correct.

**Comment by ryupold** [comment]
This commit made use of modules with the new zig build system.
It is the recommended way to link libraries.
I think the reason for the unknown type is mainly a problem of the LSP.
Not sure if reverting this change would make things really better.

**Comment by ryupold** [comment]
@profelis Sorry, I was wrong. It was actually a problem in the build.zig that caused ZLS to not be able to work properly.
I did not know that ZLS is running a build to determine the comptime stuff. At the beginning of my build.zig I expected 
an user input to select the example to run. This also blocked ZLS from compiling.

I fixed it by utilizing a build option that now can be passed like so to select an example:
`zig build run -Dexample=6`
